src/index.ts

```diff
@@ -37,6 +37,9 @@ export function createConnectorConfig(overrides?: RuntimeConfig): ConnectorRunti
         connectorConfig.modules.webhooks = _.defaultsDeep(connectorConfig.modules.webhooksV2, connectorConfig.modules.webhooks);
         delete connectorConfig.modules.webhooksV2;
     }
+    if (connectorConfig.modules.sync.enabled && connectorConfig.modules.sse.enabled) {
+        throw new Error("SSE and Sync cannot be enabled at the same time.");
```

Comment:
wdym by

```suggestion
        console.warn("The SSE and Sync modules cannot be enabled at the same time, the Sync module will be disabled.");
        connectorConfig.modules.sync.enabled = false;
```

Reply:
this is not as "harsh" as the other version" as these modules doesn't harm each other.

Reply:
Yea way better.